inst/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -1900,33 +1881,33 @@ Form
 		RadioButtonGroup
 		{
 			Layout.columnSpan:		2
-			name:					"advanced_mu_transform"
+			name:					"advancedMuTransform"
 			title:					qsTr("Transform effect sizes")
-			id:						advanced_mu_transform
-			enabled:				measures_correlation.checked || measures_OR.checked
+			id:						advancedMuTransform
+			enabled:				measuresCorrelation.checked || measuresOR.checked
 
 			RadioButton
 			{
 				label:		qsTr("log(OR)")
 				value:		"log_OR"
-				visible:	measures_OR.checked
-				id:			advanced_mu_transform_log_OR
+				visible:	measuresOR.checked
+				id:			advancedMuTransformLogOR
 			}
 
 			RadioButton
 			{
 				label:		qsTr("Cohen's d")
 				value:		"cohens_d"
```

Comment:
cohensD?

Reply:
this is passed directly to the RoBMA fitting function, I can change it to `cohensD` here, but then I would need to write another function to translate the qml input in R...

Reply:
No just keep it that way then.